src/cluster_slot_stats.c

```diff
@@ -149,8 +195,10 @@ void clusterSlotStatsCommand(client *c) {
         int desc = 1, order_by = INVALID;
         if (!strcasecmp(c->argv[3]->ptr, "key-count")) {
             order_by = KEY_COUNT;
+        } else if (!strcasecmp(c->argv[3]->ptr, "cpu-usec")) {
+            order_by = CPU_USEC;
```

Comment:
We should probably throw an error here if you pick `cpu-usec` but the extra statistics are enabled?